packages/react-experiments/src/components/UnifiedPicker/UnifiedPicker.tsx

```diff
@@ -371,6 +385,8 @@ export const UnifiedPicker = <T extends {}>(props: IUnifiedPickerProps<T>): JSX.
       selectedItemsListOnItemsRemoved,
       selection,
       showPicker,
+      selectionDeletedText,
```

Comment:
so I think that this either needs to have a callback function to get the text to announce, or just use a static text prop. If we go with the callback function, it should give the items to be deleted as params and return a string. That might be better, as it means consumers can control the behavior more. I think a static selection deleted or something else string would work too. We should probably consult someone from accessibility to see which option is better

Reply:
I like that idea. It's a lot cleaner